### DISCUSSION TITLE
Buy Limit

### DISCUSSION THREAD

**Discussion by Sylensky** [PotatoCraft-Studio/QuickShop-Reremake#234] [Ideas]
A limit for purchases each player can do would be nice.

**Comment by Gengyen** [comment]
I came here to suggest this! Players on my server will fill up a chest so at most they can only buy one stack of an item. However, some items are too rare / expensive and they end up buying more than they want.
It'd be awesome to have a command like /qs buy max <number> where the shop will not buy more than X items (or just detect when there are X of the item in it).
